stable/drone/templates/deployment-server.yaml

```diff
@@ -55,6 +55,10 @@ spec:
             value: "mysql"
           - name: DRONE_DATABASE_DATASOURCE
             value: bonanzabanzai:{{ include "mariadb.password" . }}@tcp({{ .Release.Name }}-pipeline-db:3306)/drone?parseTime=true
+
+          - name: PIPELINE_BASE_URL
+            value: http://{{ .Release.Name }}-pipeline.{{ .Release.Namespace }}:9090/pipeline/api/v1
+
```

Comment:
Isn't the base path for Pipeline API configurable? If configurable we can't have `/pipeline/` hardcoded but rather read from config.

Reply:
You're right, I will fix it.